apps/rewards/app/components/App/App.js

```diff
@@ -259,14 +269,14 @@ class App extends React.Component {
           <MyRewards
             myRewards={this.props.myRewards}
             myMetrics={this.props.myMetrics}
-            viewReward={this.viewReward}
+            viewReward={reward => this.viewReward(reward, true)}
             claimReward={this.claimReward}
           />
         ) : (
           <Overview
             rewards={this.props.rewards === undefined ? [] : this.props.rewards}
             newReward={this.newReward}
-            viewReward={this.viewReward}
+            viewReward={reward => this.viewReward(reward, false)}
```

Comment:
When looking at how a function is called, I like to have some idea of what all the arguments mean, without needing to reference the actual function definition. Passing booleans makes that really hard. What is true or false?

Fortunately, I think this is a case where using object destructuring is pretty straightforward:

```js
viewReward({ reward, isMyReward: false })